src/node/coinstats.cpp

```diff
@@ -188,33 +188,15 @@ static void FinalizeHash(MuHash3072& muhash, CCoinsStats& stats)
 }
 static void FinalizeHash(std::nullptr_t, CCoinsStats& stats) {}
 
-std::optional<CCoinsStats> LookupUTXOStatsWithIndex(CoinStatsIndex& coin_stats_index, const CBlockIndex* pindex)
-{
-    CCoinsStats stats{Assert(pindex)->nHeight, pindex->GetBlockHash()};
-
-    stats.index_used = true;
-    if (!coin_stats_index.LookUpStats(pindex, stats)) {
-        return std::nullopt;
-    }
-
-    return stats;
-}
-
-std::optional<CCoinsStats> LookupUTXOStatsWithIndex(CoinStatsIndex& coin_stats_index, CCoinsView* view, BlockManager& blockman)
```

Comment:
In commit "Move logic from LookupUTXOStatsWithIndex to CoinStatsIndex::LookUpStats" (528472c420dccbeb673dd77b64d327ab5271f23b)

This might be an example of unnecessary churn. Not a big deal, but why go through trouble of adding these LookupUTXOStatsWithIndex functions in e4b2969b54a3ddba507de1aeb17b6765ca1f102b when they will be deleted in 528472c420dccbeb673dd77b64d327ab5271f23b